converter/src/main/java/org/dswarm/converter/morph/MorphScriptBuilder.java

```diff
@@ -308,6 +310,102 @@ private void createTransformation(final Element rules, final Mapping mapping) th
 		processTransformationComponentFunction(transformationComponent, mapping, mappingInputsVariablesMap, rules);
 	}
 
+	private void createLookupTable(final Element maps, final Component transformationComponent) throws DMPConverterException {
+
+		if (transformationComponent == null)
+			return;
+
+		final Function transformationFunction = transformationComponent.getFunction();
+
+		if (transformationFunction != null && transformationFunction.getFunctionType() == FunctionType.Transformation) {
+
+			final Transformation transformation = (Transformation) transformationFunction;
+
+			final Set<Component> components = transformation.getComponents();
+
+			if (components == null)
+				return;
+
+			for (final Component component : components) {
```

Comment:
btw, do we iterate twice over the components right now, or? - I mean, it's okay, but maybe we can do all this in one iteration, or?